index.html

```diff
@@ -3,17 +3,81 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <title>Karma</title>
     <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
     <link rel="stylesheet" href="css/normalize.css">
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    <title>Karma</title>
 </head>
 <body>
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+    <header>
+        <a class="nav-list" href="index.html"><img src="/img/karma-logo.svg" alt="karma logo"></a>
+        <nav>
+            <ul class="nav-list">
+                <li class="nav-items" ><a class="nav-links" href="#">Meet Karma</a></li>
+                <li class="nav-items" ><a class="nav-links" href="#">How it Works</a></li>
+                <li class="nav-items" ><a class="nav-links" href="#">Store</a></li>
+                <li class="nav-items" ><a class="nav-links" href="#">Blog</a></li>
+                <li class="nav-items" ><a class="nav-links" href="#">Help</a></li>
+                <li class="nav-items" ><a class="nav-links" href="#">Login</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <section class="hero">
+            <h2>Introducing Karma</h2>
+            <h5>Bring WiFi with you, everywhere you go.</h5>
+            <button><a href="#">Learn More</a></button>
+        </section>
+        <section>
+            <h2>
+                Everyone needs a little Karma.
+            </h2>
+            <article>
+                <div class="text">
+                    <img src="/img/icon-devices.svg" alt="deviceicon">
```

Comment:
The alt text is not very descriptive to a user who uses a screen reader as it does not really describe the image that is being shown